src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs

```diff
@@ -38,9 +38,20 @@ public sealed class DownloadFromAzure : AzureConnectionStringBuildTask
 
         public int MaxClients { get; set; } = 8;
 
+        [Output]
+        public ITaskItem[] PathTooLongBlobs
```

Comment:
I'm curious why we are returning this. What action do we expect caller to take with these paths? Without this, the change it just the single catch (PathTooLongException) and is much simpler.... we seem to be going through a non-trivial amount of work to populate these task items in some way.

If we must return them, there should be some sort of documentation about what these task items actually contain... we seem to be populating some metadata about them, but a quick look at the task doesn't reveal that... you have to search where the backing field is being modified to learn what properties the returned items have. I still personally vote for just removing the output altogether, since there isn't really anything the caller can do.